docs/src/literate-tutorials/dg_heat_equation.jl

```diff
@@ -0,0 +1,411 @@
+# # [Discontinuous Galerkin Heat equation](@id tutorial-dg-heat-equation)
+#
+# ![](dg_heat_equation.png)
+#
+# *Figure 1*: Temperature field on the unit square with an internal uniform heat source
+# solved with inhomogeneous Dirichlet boundary conditions on the left and right boundaries and flux on the top and bottom boundaries.
+#
+#-
+#md # !!! tip
+#md #     This example is also available as a Jupyter notebook:
+#md #     [`dg_heat_equation.ipynb`](@__NBVIEWER_ROOT_URL__/examples/dg_heat_equation.ipynb).
+#-
+#
+# This example was developed
+# as part of the Google summer of code funded project
+# ["Discontinuous Galerkin Infrastructure For the finite element toolbox Ferrite.jl"](https://summerofcode.withgoogle.com/programs/2023/projects/SLGbRNI5)
+#
+# ## Introduction
+#
+# The heat equation is the "Hello, world!" equation of finite elements.
+# Here we solve the equation on a unit square with a uniform internal source, using the interior penalty formulation.
+# The strong form of the (linear) heat equation is given by
```

Comment:
Would it make sense to (1) link to the basic example here and (2) point out the differences?